Assets/MixedRealityToolkit.Extensions/QRCodesTracker/Scripts/QRCodesManager.cs

```diff
@@ -89,8 +87,17 @@ public bool TryGetLocationForQRCode(Guid id, out Matrix4x4 location)
             location = Matrix4x4.identity;
 
 #if WINDOWS_UWP
-            var coordinateSystem = SpatialGraphInteropPreview.CreateCoordinateSystemForNode(id);
-            return TryGetLocationForQRCode(coordinateSystem, out location);
+            try
+            {
+                var coordinateSystem = SpatialGraphInteropPreview.CreateCoordinateSystemForNode(id);
+                return TryGetLocationForQRCode(coordinateSystem, out location);
+            }
+            catch (Exception e)
+            {
+                Debug.LogError($"Exception thrown creating coordinate system for qr code id: {e.ToString()}");
+                return false;
+            }
+
 #else
             Debug.LogError("Failed to create coordinate system for qr code id");
```

Comment:
>id" [](start = 75, length = 3)

Nit: Unrelated to this PR, but might be helpful to list the ID here (it seems like it was intended to list the ID but maybe it was missed)